approval_frame/templates/registration/password_reset_form.html

```diff
@@ -0,0 +1,41 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <div class='row-fluid top-buffer'>
+      <div class='col-md-6 col-md-offset-3'>
+        <legend>Reset Password</legend>
+        <p>Enter your email address below and we will send you an email with
+         instructions to reset your password.</p>
+        <form action="." method="post">
+        {% csrf_token %}
+          <div class='form-group {% if form.errors.email %} has-error {% endif %}'>
+            {% for error in form.errors.email %}
+              <label class='control-label' for='{{ form.email.id_for_label }}'>
+                {{ error }}
+              </label>
+            {% endfor %}
+            <input type='text' class='form-control' id='{{ form.email.id_for_label }}'
+                  name='{{ form.email.name }}' placeholder='Email ID'
+                  maxlength={{ form.email.field.max_length }}>
+          </div>
+          <button type='submit' class='btn btn-success'>Submit</button>
+         </form>
```

Comment:
I get it now, the closing form tag was only one space to the left of the button tag above it, so that confused me. Everything is in the right place except for the closing form tag.